server/expression/filters/length.go

```diff
@@ -0,0 +1,26 @@
+package filters
+
+import (
+	"fmt"
+
+	"github.com/kubeshop/tracetest/server/expression/types"
+)
+
+func Length(input Value, args ...string) (Value, error) {
+	if len(args) != 0 {
+		return Value{}, fmt.Errorf("wrong number of args. Expected 0, got %d", len(args))
+	}
+
+	if len(input) != 1 {
+		return Value{}, fmt.Errorf("wrong input: Expected string, not an array")
```

Comment:
could we make it work with arrays too? like `["1", "2"] | length = 2`

Reply:
Done. Merged `count` and `length` together. Now `length` works with arrays and strings.